tests/parser/tstrongspaces.nim

```diff
@@ -27,7 +44,7 @@ when true:
   let b = 66
   let c = 90
   let bar = 8000
-  if foo+4 * 4 == 8 and b&c | 9  ++
+  if foo+4 * 4 == 8  and  b&c | 9  ++
```

Comment:
For reference: http://nim-lang.org/manual.html#precedence

I considered not applying the strongSpaces rule to keyword operators with priority lower than the comparison operators (lower than 5). That would be `and` `or` `xor`. The problem is that then `@ : ?``operators and arrow like operators would have always a priority higher than them, changing the priority rules. Then I settled with this consistent rule, applying it to everything.

Another possibility is to apply the strongSpaces rule only to arithmetic operators. For all precedence levels 5/6/7 (not sure where to start) and above, and only them. This would be a bigger change, but maybe more what you imagine being ideal.

By the way, in the docs the arrow like operator has precedence 0, but in the code it has precedence 1, and gets higher precedence than assignment operators when strongSpaces is on. This is a bug?